developer/src/kmc-ldml/test/layr.tests.ts

```diff
@@ -107,13 +108,13 @@ describe('layr', function () {
     {
       // missing layer element
       subpath: 'sections/layr/invalid-missing-layer.xml',
-      errors: [LdmlCompilerMessages.Error_MustBeAtLeastOneLayerElement(withOffset(258))],
+      errors: [LdmlCompilerMessages.Error_MustBeAtLeastOneLayerElement(withOffset(40))],
```

Comment:
The offset changed here?

Reply:
i think it was an error in the test

Reply:
https://github.com/keymanapp/keyman/pull/14104/files#diff-6368eb7cd37506920e715122a3ec3a946faa516c43ab31b816cdb672bed1cb99R11

oh, it was because the file was actually invalid before, but with additional validation the fixture needed update